peachjam/templates/peachjam/_case_histories.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>{% trans 'Case' %}</th>
```

Comment:
Please put the date first, so that it works as a timeline going back in time.

Put the outcome last.

Reply:
done